### PR TITLE
added age info

### PR DESCRIPTION
**Pull request by kenta3**
i am actually 19 yo.

### CODE REVIEW

**Comment by kenta3** [comment]
test_comment_0

**Comment by kenta3** [comment]
test_comment_1

File: main.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
 print('hello world')
```
Comment (fukano0401) [inline main.py]: this is review comment.

**Comment by fukano0401** [comment]
comment by fukano.

**Comment by kenta3** [comment]
comment 28

**Pull request merged**